src/main/java/uk/gov/pay/api/filter/ratelimit/RateLimiter.java

```diff
@@ -12,50 +13,51 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RateLimiter.class);
 
-    private final int rate;
-    private final int rateForPost;
+    private int rate;
+    private int rateForPost;
+    private int auditRate;
+    private int perMillis;
+    private int noOfNodes;
     
     private static final String METHOD_POST = "POST";
```

Comment:
"POST" is already defined in javax.ws.rs.HttpMethod.POST, can we use that please.

Reply:
Thank you, changed to HttpMethod.POST